getting-started.md

```diff
@@ -59,3 +59,23 @@ $ exogress spawn --access-key-id=<ACCESS_KEY_ID> --secret-access-key=<SECRET_ACC
 ```
 
 That's it. Now the server on port `3000` is available under the selected domain.
+
+## Advanced
```

Comment:
i wouldn't call it an "Advanced" topic

Reply:
just leave in a general "Getting started"? my point was that these items don't require immediate setup